apps/www/src/content/docs/components/navigation-menu.md

```diff
@@ -56,10 +56,8 @@ import { navigationMenuTriggerStyle } from '@/components/ui/navigation-menu'
 ```vue
 <template>
   <NavigationMenuItem>
-    <NuxtLink to="/docs">
-      <NavigationMenuLink :class="navigationMenuTriggerStyle()">
```

Comment:
We need `NavigationMenuLink` here. We can set this to `as="span"` so it renders as inline element. And the upstream issue has been fixed https://github.com/radix-vue/radix-vue/issues/1196